main.py

```diff
@@ -244,10 +244,14 @@ def command_unknown(self, update: Update, context: CallbackContext) -> None:
 
     def command_list_devices(self, update: Update, context: CallbackContext) -> None:
         device_groups = self.get_user(update).device_groups
+        device_filter = ' '.join(context.args).lower()
         devices = dict()
         for device_group in device_groups:
             for device in device_group.get_devices().values():
-                devices[device['label']] = device
+                name = device['label']
+                # lower(): Hack because Python doesn't support case-insensitive searches
+                if not device_filter or device_filter in name.lower():
```

Comment:
Can be simplified to `if device_filter in name.lower():`
```
>>> ' '.join([]).lower()
''
>>> '' in "foo"
True
>>> 'foo' in 'barfoo'
True
```

Reply:
ok